content/rc/administration/setup/edit-subscription.md

```diff
@@ -57,7 +57,7 @@ communicate with each other as if they are within the same network. You can conn
 VPC in the Redis Cloud subscription to the VPC of your application. Then your application can
 connect securely to your Redis Cloud database using VPC Peering to optimize the performance of your application.
 
-The VPC peering configuration requires you to initiate VPC peering on your Redis Cloud Pro or Ultimate subcription
+The VPC peering configuration requires you to initiate VPC peering on your Redis Cloud <!-- Pro or Ultimate --> subscription
```

Comment:
Why are you using html comments here?

Reply:
Stubbing out temporarily, pending further discussion with PM re: terminology.